README.md

```diff
@@ -121,3 +121,22 @@ mvn release:prepare release:clean -Darguments=-DskipTests
 ```
 
 With commands you are able to define a release, a tag and a next development version name in interactive way.
+
+## Issues found by the testsuite
+Any issue found in WildFly/EAP release will be reported into the JBoss issue tracker: https://issues.redhat.com/ . There are the following Jira projects:
```

Comment:
Sorry to pointing this out again, but the issue tracker itself was renamed from "JBoss issue tracker" to "Red Hat issue tracker" too...